pallets/loans/src/lib.rs

```diff
@@ -1035,17 +1036,17 @@ impl<T: Config> Pallet<T> {
 
     // Calculates and returns the most recent amount of borrowed balance of `currency_id`
     // for `who`.
-    pub fn borrow_balance_stored(
+    pub fn recent_borrow_balance(
```

Comment:
The name refactor doesn't make sense, `recent` is too general.

Reply:
do you have an suggestion here? like `latest`

Reply:
Maybe `current`